src/dmd/traits.d

```diff
@@ -1973,7 +1973,17 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
             for (auto ns = p.cppnamespace; ns !is null; ns = ns.cppnamespace)
             {
                 ns.dsymbolSemantic(sc);
-                exps.insert(0, ns.exp);
+
+                if (ns.exp.isErrorExp())
+                    return ErrorExp.get();
+
+                auto se = ns.exp.toStringExp();
```

Comment:
Is it possible for this to return `null`?

Reply:
its has gone through semantic at this point so I don't think so.